pkg/resources/factory.go

```diff
@@ -119,3 +130,7 @@ func (rf *resourceFactory) GetResourcePool(rc *types.ResourceConfig, deviceList
 	rPool := newResourcePool(rc, apiDevices, devicePool)
 	return rPool, nil
 }
+
+func (rf *resourceFactory) GetRdmaSpec(vendor, pciAddrs string) types.RdmaSpec {
```

Comment:
@Mmduh-483  remove the vendor you are not using it anyway

Reply:
Ok, lets remove the "vendor" from this method's parameter for now. Once we get it verified by @zshi-redhat again that it works with all the new changes we can go ahead and merge it.
But please keep in mind we may need changes in future to facilitate other modes of RDMA.

Reply:
done